crates/viewer/re_viewer_context/src/viewer_context.rs

```diff
@@ -86,11 +89,8 @@ impl ViewerContext<'_> {
 
     /// Runtime info about components and archetypes.
     ///
-    /// The component placeholder values for components are to be used when [`crate::ComponentFallbackProvider::try_provide_fallback`]
-    /// is not able to provide a value.
-    ///
-    /// ⚠️ In almost all cases you should not use this directly, but instead use the currently best fitting
-    /// [`crate::ComponentFallbackProvider`] and call [`crate::ComponentFallbackProvider::fallback_for`] instead.
+    /// ⚠️ In almost all cases you should not use this directly, but instead use
+    /// [`crate::FallbackProviderRegistry::fallback_for`].
```

Comment:
guess we can remove that entirely since by now the reflection is used for all sort of things not just getting defaults